source/assets/stylesheets/styles/_blog_large_tile.scss

```diff
@@ -130,6 +130,8 @@
   display: block;
   height: 130px;
   object-fit: cover;
+  //requires object-fit-fix/ofi.min.js to be loaded
```

Comment:
Can we add a mini-comment about why this is needed in the same way as we did last time? Just a note that it's needed for IE on the same line is all. Just hopefully a little favour to the next dev if they're not aware of this library.